README.md

```diff
@@ -1,70 +1,127 @@
+but via either the [`oc adm upgrade`][ocp-oc-adm-upgrade] CLI or the [web console][ocp-webconsole-upgrades].
 
-1. Run the following command to get render the release-payload contents to `/tmp/cvo`
+## Understanding Upgrades
 
-```sh
-podman run --rm -ti \
-    -v /tmp/cvo:/tmp/cvo:z \
-    <release image> \
-        render \
-        --output-dir=/tmp/cvo \
-        --release-image="<release image>"
-```
+> :bulb: This section is only a high-level overview. See the [Update Process][dev-guide-upgrade-workflow] and
+> [Reconciliation][dev-guide-reconciliation] documents in the [dev-guide][dev-guide] for more details.
 
-`<release image>` can be personal release image generated using [this](#building-release-image-using-local-cvo) or Origin's release image like `registry.ci.openshift.org/openshift/origin-release:v4.0`.
+The Cluster Version Operator continuously fetches information about upgrade paths for the configured channel from the
+OpenShift Update Service (OSUS). It stores the recommended update options in the `status.availableUpdates` field of
+its `ClusterVersion` resource.
 
-## Running CVO tests
+The intent to upgrade the cluster to another version is expressed by storing the desired version in
+the `spec. desiredUpdate` field. When `spec.desiredUpdate` does not match the current cluster version, CVO will start
+updating the cluster. It downloads the release payload image, validates it, and [systematically reconciles]
+[dev-guide-reconciliation] the Cluster Operator resources to match the updated manifests delivered in the release
+payload image.
 
-```sh
-# Run all unit tests
-go test ./...
+## Troubleshooting
 
-# Run integration tests against a cluster (creates content in a given namespace)
-# Requires the CVO CRD to be installed.
-export KUBECONFIG=<admin kubeconfig>
-TEST_INTEGRATION=1 go test ./... -test.run=^TestIntegration
+A typical OpenShift cluster will have a `Deployment` resource called `cluster-version-operator` in
+the `openshift-cluster-version` namespace, configured to run a single CVO replica. Confirm that its `Pod` is up and
+optionally inspect its log:
+
+```console
+$ oc get deployment -n openshift-cluster-version cluster-version-operator
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+cluster-version-operator   1/1     1            1           2y227d
+
+$ oc get pods -n openshift-cluster-version -l k8s-app=cluster-version-operator
+NAME                                        READY   STATUS    RESTARTS   AGE
+cluster-version-operator-7fdf58f594-ct92x   1/1     Running   1          85d
+
+$ oc logs -n openshift-cluster-version -l k8s-app=cluster-version-operator
+...
 ```
 
-## Contributing
+The CVO follows the [Kubernetes API conventions][kube-api-conventions-status] and sets `Conditions` in the status of
+its `ClusterVersion` resource. These conditions are surfaced by both the OpenShift web console and the `oc adm-cli`.
```

Comment:
nit: `oc adm-cli` reads like it's suggesting a specific invocation, when I think you mean either `oc adm upgrade` or "the `oc adm ...` CLI" or some such.